pkg/log/log.go

```diff
@@ -0,0 +1,89 @@
+package log
+
+import (
+	"log"
+)
+
+// Based partially on the following links:
+// https://github.com/go-kit/kit/blob/master/log/level/level.go
+// https://www.ardanlabs.com/blog/2013/11/using-log-package-in-go.html
+var Logger ResourceLogger
+
+type ResourceLogger struct {
+	Logger *log.Logger
+	Level  int
+	ready  bool
+}
+
+func (rl *ResourceLogger) InitLoggerFromParam(levelStringValue string) {
+	rl.InitLogger(GetValueFromParam(levelStringValue))
+}
+
+func (rl *ResourceLogger) InitLogger(levelValue int) {
+	if levelValue < MinLevel || levelValue > MaxLevel {
+		rl.Level = INFO
+	} else {
+		rl.Level = levelValue
+	}
+
+	rl.Logger = log.New(GetIOHandle(rl.Level), GetPrefixForLogger(rl.Level),
+		log.Ldate|log.Ltime)
+
+	rl.ready = true
+}
+
+func (rl *ResourceLogger) Debug(vals ...interface{}) {
+	rl.log(DEBUG, vals)
+}
+
+func (rl *ResourceLogger) Debugf(format string, vals ...interface{}) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method ResourceLogger.Debugf should have comment or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)